CollAction.Tests/MailChimpManagerTests.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using CollAction.Helpers;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+
+namespace CollAction.Tests
+{
+    [TestClass]
+    public class MailChimpManagerTests
+    {
+        private readonly string newsletterListId = "2141ab4819"; // The ID for our "CollAction Newsletter" list on MailChimp.
```

Comment:
Is this used for the tests: https://developer.mailchimp.com/documentation/mailchimp/guides/about-the-api-playground/